node-runner-cli/test-prompts/gateway-local-core-remote-postgress.yml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
currently not. I tried with the docker container. But with no success. I did not want to delete the prompts nor the bash script yet though, because they are still useful for testing manually or using external services.